db/sql/sql_test.go

```diff
@@ -353,8 +360,189 @@ var _ = Describe("Sql", func() {
 			})
 		})
 	})
+
+	Describe("Query construction", func() {
+		var (
+			query         squirrel.SelectBuilder
+			expectedQuery squirrel.SelectBuilder
+		)
+		BeforeEach(func() {
+			t := testSchema.GetDbTableName()
+			query = squirrel.Select("*").From(quote(t))
+			expectedQuery = query
+		})
+		Context("Basic select query", func() {
+			It("should create empty select query", func() {
+				filter := map[string]interface{}{}
+
+				res, err := AddFilterToQuery(testSchema, query, filter, false)
+
+				Expect(err).ToNot(HaveOccurred())
+				resSql, param, err := res.ToSql()
+				Expect(err).ToNot(HaveOccurred())
+				expectedSql, expectedParam, _ := expectedQuery.ToSql()
+				Expect(resSql).To(Equal(expectedSql))
+				Expect(param).To(Equal(expectedParam))
+			})
+			It("should create select query with one parameter", func() {
+				filter := map[string]interface{}{"test_string": "123"}
```

Comment:
is it possible to express `test_string != "123"` using the new API? how to do it? do we need to do sth like `{"__and__": {"property":"test_string", "type":"neq", "value":123}}`?
please add a test case for it

Reply:
exactly, added test